packages/framework/src/Form/Admin/Category/CategoryFormType.php

```diff
@@ -234,20 +233,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'info_text' => t('You can upload following formats: PNG, JPG, GIF'),
             ]);
 
-        $builderPluginGroup = $builder->create('plugin', FormType::class, [
-            'inherit_data' => true,
-            'is_plugin_data_group' => true,
```

Comment:
is this option relevant yet? It was used only here (and it was a bad usage if I get it correctly) so I think it would be nice to remove `is_plugin_data_group` option completely from code and docs, what do you think?

Reply:
Hello @vitek-rostislav,
yes you are right, this option is no longer used.

Reply:
Are you sure, guys? See 
https://github.com/shopsys/shopsys/blob/master/docs/introduction/using-form-types.md#is_plugin_data_group

You might be thinking of `is_group_container_to_render_as_the_last_one`...?

Reply:
OK, my bad - you remove the last usage of the option. It is used in the template, but the boolean option is always `false` now. Thanks @TomasLudvik for clarification.

Removal of the option should be considered a BC break imo (even though we never defined BC in forms as described in #963). Maybe we can mark the option as deprecated and remove it in `8.0`?

Reply:
I do not know how to mark option in form type as deprecated :(

Reply:
@TomasLudvik I've tried to mark it as deprecated in c2c3ad2af in similar fashion as in Symfony code. When I've configured a form type with `is_plugin_data_group = true`, a deprecation notice appeared in the debug toolbar as expected.

Reply:
It might be even better to trigger the deprecation error when `false` is explicitly provided (which means the default option had to be removed, otherwise it would always trigger the error). See 5d49dc7e4.

Now, when no such deprecations are triggered on a project, the option may be in fact safely removed.